### ISSUE TITLE
Some sounds repeat last half-second when played.

### ISSUE THREAD

**Issue by KevinNovak** [DreamNetwork/gmod-tauntpack#2]
See https://forum.facepunch.com/gmoddev/buudo/EmitSound-on-player-glitches-repeats-at-the-end-of-the-sound/1/ for more information.

Some sounds after downloaded from the [Workshop](https://steamcommunity.com/sharedfiles/filedetails/?id=426548393) are duplicating the last half-second or so of audio when played. This only happens with the workshop sounds, not when directly downloaded from the GitHub. 

**Discovered problem sounds:**
- taunts/dreamnetwork/props/dn0038.ogg 
- taunts/dreamnetwork/props/ih0022.mp3
- taunts/dreamnetwork/props/dn0041.ogg
- taunts/dreamnetwork/props/dn003E.ogg
- taunts/dreamnetwork/props/dn003D.ogg

There may be more, I haven't checked all of them.

**Example:**
_taunts/dreamnetwork/props/dn0038.ogg (Airhorn)_
![Example](https://i.imgur.com/K0tW2L7.png)
The first sample is from the Github.
The second sample is from the Workshop.

Both sound normally when played through Windows Media Player and VLC, but the workshop file duplicates the end in Garrys Mod and the duplication is viewable in Audacity.

**This leads me to believe:**
- The files in the Github don't match the workshop
- Something in uploading the file to the Steam Workshop corrupted the files.
- Something in downloading the file from the Workshop corrupted the files.
- Something in retrieving the sounds from the GMA corrupted the files.

**Comment by KevinNovak** [comment]
I did some tests, and it seems like uploading back in 2015 corrupted some of the sound files, but reuploading should fix the issue.

I did 3 tests:
1. Downloaded this GitHub repo and packed into a GMA. Extracted GMA. Sound file was not corrupted.
2. Uploaded the packed GMA to steam workshop. Subscribed. Extracted GMA. Sound file was not corrupted.
3. Subscribed to existing workshop item from 2015 (https://steamcommunity.com/sharedfiles/filedetails/?id=426548393). Extracted GMA. Sound file was corrupted.

![Tests](https://i.imgur.com/b2bc7gT.png)

**TLDR**: Reuploading to the workshop should fix the issue.

**Comment by icedream** [comment]
Thank you very much for the detailed analysis of the issue! I'll try to get to this asap but I'm currently loaded with a bunch of other projects to work on so it may take a while.

**Comment by icedream** [comment]
I'll shove this together with #1 since by the time I get to this I'll probably just try and get this taunt pack updated overall.

**Comment by icedream** [comment]
It's been a while but I've managed to do a repush of the last version.
